ML_End_to_End/StartingCNN.m

```diff
@@ -0,0 +1,37 @@
+% Setting up CNN Layers
+
+clc
+clear
+close all
+
+% Loading from Kaylee
+filepath = "C:\Users\kyler\OneDrive\School\Colorado State\2023\MATH 633\MATH633_MedicalImaging\AnatomicalAtlas\case-104862\voltages\voltage_files_run1";
```

Comment:
Is there value to using fileparts(mfilename('fullpath)) to get the location of this script on the machine it is running on and then define the filepaths relative to that so it could run on other machines? Will we want/need this to be run by anybody else?

Reply:
If you are referring to line 8, that was just a temp file so we could have something started easy. This program will be combined with Kaylee's and then that line will be completely removed

Reply:
Roger